sdcm/cluster.py

```diff
@@ -4598,6 +4599,9 @@ def _scylla_install(self, node):
             if ":" in scylla_repo.rsplit("/", 1)[-1]:
                 scylla_repo, version = scylla_repo.rsplit(":", 1)
             node.install_scylla(scylla_repo=scylla_repo, scylla_version=version)
+        # remove dependency when using scylla ami but installing from repo
```

Comment:
I would rather put this kind of thing under some flag, for this case, and not to be done after every installation

Reply:
I didn't consider it harmful, so for sake of simplicity put it this way.
Do you find it important?

Reply:
it changes something in the installation, if we are going to do it alway, why it's there from the first place.
if it has some purpose we shouldn't be removing it by default.

so I don't know if it's important or not, and rather not find out it is, and our upgrade test might be missing things that might have to users, cause of we remove it.

I rather not take that risk.